drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig QSPI_NOR
+	bool "QSPI NOR Flash"
+	select FLASH_HAS_DRIVER_ENABLED
+	depends on HAS_HW_NRF_QSPI
+
+if QSPI_NOR
```

Comment:
Add a blank line after the `if` to make it balanced with the `endif`. See https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#header-comments-and-other-nits.